feat(changelog): documentation-added-the-sdk-js-1.0-has 2023-02-23 (#1342)
scaleway/docs-content: changelog/february2023/2023-02-23-documentation-added-the-sdk-js-1.0-has.mdx

---

> title: The SDK-JS 1.0 has been released

Reviewer: ```suggestion
title: The SDK-JS 1.0 has been released!
```

> title: The SDK-JS 1.0 has been released

Reviewer 2: ```suggestion
title: The SDK-JS 1.0 is now available
```

status: added
author:
    fullname: 'Join the #sdk-js channel on Slack.'
    url: 'https://slack.scaleway.com'

> date: 2/23/2023

Reviewer 2: ```suggestion
date: 2023/02/23
```

category: documentation
product: documentation
---

> The JavaScript SDK is out of beta! It enables you to interact with Scaleway APIs. For more information and follow the updates, don't hesitate to take a look at the [GitHub repository](https://github.com/scaleway/scaleway-sdk-js).

Reviewer 2: ```suggestion
The JavaScript SDK is now available! It allows you to interact with Scaleway APIs. For more information and updates, check out the [dedicated GitHub repository](https://github.com/scaleway/scaleway-sdk-js).
```
